main/src/cgeo/geocaching/enumerations/CacheType.java

```diff
@@ -36,7 +36,7 @@
     GPS_EXHIBIT("gps", "GPS Adventures Exhibit", "72e69af2-7986-4990-afd9-bc16cbbb4ce3", R.string.gps, R.drawable.type_event, "1304", R.drawable.dot_event), // icon missing
     BLOCK_PARTY("block", "Geocaching HQ Block Party", "bc2f3df2-1aab-4601-b2ff-b5091f6c02e3", R.string.block, R.drawable.type_event, "4738", R.drawable.dot_event), // icon missing
     LOCATIONLESS("locationless", "Locationless (Reverse) Cache", "8f6dd7bc-ff39-4997-bd2e-225a0d2adf9d", R.string.locationless, R.drawable.type_virtual, "12", R.drawable.dot_virtual), // icon missing
-    ADVLAB("advlab", "Adventure Lab Cache", "", R.string.advlab, R.drawable.type_advlab, "", R.drawable.dot_advlab),
+    ADVLAB("advlab", "Lab Cache", "", R.string.advlab, R.drawable.type_advlab, "", R.drawable.dot_advlab), // pattern "Lab Cache" is used by https://gcutils.de/lab2gpx/
```

Comment:
Doesn't that change the cache type name also for the LC connector?

Reply:
The user displayable string is represented by "R.string.advlab". This one is only for parsing...